litmus/director/pool-upgrade/run_litmus_test.yml

```diff
@@ -0,0 +1,62 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: pool-upgrade-check
```

Comment:
Put a `-` after openebs-install so that the object is created as `openebs-install-xxxx` else it will be created as `openebs-installxxxx` which will not be appropriate.